dao/mongodb.rya/src/main/java/mvm/rya/mongodb/instance/MongoDetailsAdapter.java

```diff
@@ -180,20 +177,16 @@ public static RyaDetails toRyaDetails(final DBObject mongoObj) throws MalformedR
         }
     }
 
+    /**
+     * @param basicObj
+     * @return A builder setting the PCJIndexer to false since Mongo does not support PCJ indexing.
+     */
     private static PCJIndexDetails.Builder getPCJIndexDetails(final BasicDBObject basicObj) {
         final BasicDBObject pcjIndexDBO = (BasicDBObject) basicObj.get(PCJ_DETAILS_KEY);
 
         final PCJIndexDetails.Builder pcjBuilder = PCJIndexDetails.builder()
```

Comment:
Revert this change too. Even though we don't currently support PCJs, I think that nuance should be encapsulated elsewhere.